src/js/weekly-trends.js

```diff
@@ -88,6 +94,11 @@ document.addEventListener('DOMContentLoaded', async function () {
         card.addEventListener('click', event => {

```

Comment:
event nieużywany

Reply:
Nie wiem jak to jest, widze że niby nie używany ale jak to usuwam to Weekly trends klikanie i otwieranie modala nie działa :P

Reply:
event podmień na ()
powinno działać

Reply:
Zmieniłam i poprawiłam - działa wszystko.